Makefile

```diff
@@ -25,25 +39,18 @@ default: docs
 .PHONY: docs
 docs: output/docs.pdf output/docs.html
 
+ifeq "$(strip $(PANDOC))" ''
+output/docs.pdf output/docs.html:
```

Comment:
Sure.  Do you want oci-runtime-spec-v{version}.{extension}?  If we're adding context in the filename, we might as well be thorough ;).